.github/workflows/maven.yml

```diff
@@ -0,0 +1,19 @@
+name: APIM Product Build
+
+on:
+  schedule:
+  - cron: 0 5/15/23 * * *
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
can we make this 11

Reply:
planning to add more jobs to cover all combinations. let's start with 8.